src/test/scala/org/broadinstitute/dsde/agora/server/webservice/CompatibleConfigurationIntegrationSpec.scala

```diff
@@ -30,80 +30,123 @@ class CompatibleConfigurationIntegrationSpec extends FreeSpec with ExecutionDire
   val methodsService = new MethodsService(permsDataSource) with ActorRefFactoryContext
   val testRoutes = ApiService.handleExceptionsAndRejections (methodsService.queryCompatibleConfigurationsRoute)
 
+  // The following methods create a method AgoraEntity while at the same time constructing the Mock Waas
+  // describe response.  This test is more difficult than the others because it generates many unique
+  // method payloads (WDLs) so it can test the compatibility of methods
```

Comment:
and, we define "compatibility" in this context as having matching inputs/outputs, which requires waas parsing!

Reply:
added